src/cmd/resources/CMakeLists.erb

```diff
@@ -0,0 +1,80 @@
+#These lines will set minimum required version of cmake and declare the name of the project.
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+project(<%= options['name'] %>)
+<% if options['ign_dependecies']==1 %>
+
+#This is a list of general packages on which your ignition package will depend,find_package function will find and source theses packages.
+#You can add or delete dependencies based on your requirement.
+<% if options['built_type']=='ament_cmake' %>
+#Since,the package is amnet dependent we will find and call ament_cmake.
+
+find_package(ament_cmake REQUIRED)
+<% end %>
+
+find_package(ignition-cmake<%= dependencies_dict["ign-cmake"]%> REQUIRED)
+
+find_package(ignition-gazebo<%= dependencies_dict["ign-gazebo"]%> REQUIRED COMPONENTS gui)
+set(IGN_GAZEBO_VER ${ignition-gazebo<%= dependencies_dict["ign-gazebo"]%>_VERSION_MAJOR})
+
+find_package(ignition-gui<%= dependencies_dict["ign-gui"]%> REQUIRED)
+set(IGN_GUI_VER ${ignition-gui<%= dependencies_dict["ign-gui"]%>_VERSION_MAJOR})
+
+find_package(ignition-rendering<%= dependencies_dict["ign-rendering"]%> REQUIRED)
```

Comment:
TODO: we need to handle the change for garden where everything is `gz-`

Reply:
Sure,once we finish this in fortress we can see for that